client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -37,9 +38,11 @@ import {
 	isBusiness,
 	isEnterprise
 } from 'lib/products-values';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isAutomatedTransferActive } from 'state/selectors';
 import QueryEligibility from 'components/data/query-atat-eligibility';
 
-export default React.createClass( {
+const PluginMeta = React.createClass( {
 	OUT_OF_DATE_YEARS: 2,
 
 	displayName: 'PluginMeta',
```

Comment:
Giving a name here to the React class means `displayName` is redundant. Might as well remove it